english/robofriend/cuddleattack/AIattacks.md

```diff
@@ -0,0 +1,9 @@
+You've unpacked a CloneFriend!
+You feel uneasy, 
+holding this little robot in your hand, 
+knowing it's capabilities. 
+It begins a countdown to activate: 
+5 - 4 - 3 - 2 - 1 ....
+AND YOU'VE BEEN CUDDLE-ATTACKED BY YOUR CloneFriend! *hugs* 
+
+You'll never be alone again. [(If that doesn't scare you, better go back)](../marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.